insight/admin/weekly_trend_admin.py

```
@@ -50,12 +56,25 @@ class WeeklyTrendAdmin(admin.ModelAdmin, BaseTrendAdminMixin):
         (
             "처리 상태",
             {
-                "fields": ("is_processed", "processed_at"),
+                "fields": ("review_status", "is_processed", "processed_at"),
             },
         ),
     )
 
-    actions = ["mark_as_processed"]
+    actions = ["mark_as_processed", "mark_as_hold", "mark_as_approved"]
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)